aea/agent.py

```diff
@@ -290,3 +295,37 @@ def exception_handler(
             f"Exception {repr(exception)} raised during {repr(function)} call."
         )
         return True
+
+
+class SkillOutBox(OutBox):
```

Comment:
The concept of skill does not exist on the `Agent` level, should be added on `AEA` level